Sming/Libraries/Adafruit_ILI9341/Adafruit_ILI9341.cpp

```diff
@@ -157,7 +157,8 @@ void Adafruit_ILI9341::begin(void) {
 	transmitCmdData(0xE1, data, 15);    	//Set Gamma
 
 	transmitCmd(0x11);    	//Exit Sleep
-	os_delay_us(120000);
```

Comment:
More code should be reviewed then to check if delays do not excede 65,535 limit.